lib/items.ts

```diff
@@ -2,43 +2,45 @@ import { Item, PlayedItem } from "../types/item";
 import { createWikimediaImage } from "./image";
 
 export function getRandomItem(deck: Item[], played: Item[]): Item {
-  const playedYears = played.map((item): number => {
-    return item.year;
-  });
-
   const periods: [number, number][] = [
-    [-100000, 1400],
-    [1400, 1850],
-    [1850, 1930],
-    [1930, 2020],
+    [-100000, 1000],
+    [1000, 1800],
+    [1800, 2020],
   ];
   const [fromYear, toYear] =
     periods[Math.floor(Math.random() * periods.length)];
   const avoidPeople = Math.random() > 0.5;
-
   const candidates = deck.filter((candidate) => {
     if (avoidPeople && candidate.instance_of.includes("human")) {
       return false;
     }
-
     if (candidate.year < fromYear || candidate.year > toYear) {
       return false;
     }
-
-    if (playedYears.includes(candidate.year)) {
+    if (tooClose(candidate, played)) {
       return false;
     }
-
     return true;
   });
 
-  if (candidates.length === 0) {
-    throw new Error("No item candidates");
+  if (candidates.length > 0) {
+    return candidates[Math.floor(Math.random() * candidates.length)];
   }
+  throw new Error("No item candidates");
+  return deck[Math.floor(Math.random() * deck.length)];
```

Comment:
This will never be run because the previous line throws an error. I think you should just remove the `throw`?

Reply:
Oh, I didn't realize that throw would end the function.  I've removed it.  Thanks!